internal/test/wsu/wsu_test.go

```diff
@@ -285,10 +458,38 @@ func testHNSNetworksCreated(t *testing.T, vm e2ef.WindowsVM) {
 		"Could not find OpenShiftNetwork in list of HNS Networks")
 }
 
+// getAffinityForNode returns an affinity which matches the associated node's name
+func getAffinityForNode(externalIP string) (*v1.Affinity, error) {
+	node, err := getNode(externalIP)
```

Comment:
Here again, you don't need to make the `getNode` call but rather pass the node object into the function.